packages/wdio-browserstack-service/src/data-store.ts

```diff
@@ -0,0 +1,51 @@
+import path from 'node:path'
+import fs from 'node:fs'
+
+import { BStackLogger } from './bstackLogger.js'
+
+class DataStore {
```

Comment:
There is no reason to have a class for this if all methods of the class are `static`. Let's just have basic functions that are being exported here.

Reply:
Have removed class and made them normal functions https://github.com/webdriverio/webdriverio/pull/12392/commits/a498f3c8ed156f3c4b7ae5a91722f103795d5898